code/obj/item/satchel.dm

```diff
@@ -112,8 +112,9 @@
 
 	MouseDrop_T(atom/movable/O as obj, mob/user as mob)
 		var/proceed = 0
+		var/obj/item/W = O
 		for(var/check_path in src.allowed)
-			if(istype(O, check_path))
+			if(istype(O, check_path) && W.w_class < 4)
```

Comment:
Will runtime.

Reply:
couldn't reproduce runtime ingame regardless of any combination. Moved the define back to where it originally was